ext/spl/spl_fixedarray.c

```diff
@@ -358,6 +358,20 @@ static zval *spl_fixedarray_object_read_dimension(zval *object, zval *offset, in
 
 	intern = Z_SPLFIXEDARRAY_P(object);
 
+	if (type == BP_VAR_IS) {
```

Comment:
Though you might want to simply delegate to spl_fixedarray_object_has_dimension here.